x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/hooks/use_create_analytics_form/state.ts

```diff
@@ -290,7 +291,7 @@ export function getFormStateFromJobConfig(
   analyticsJobConfig: Readonly<CloneDataFrameAnalyticsConfig>,
   isClone: boolean = true
 ): Partial<State['form']> {
-  const jobType = Object.keys(analyticsJobConfig.analysis)[0] as ANALYSIS_CONFIG_TYPE;
+  const jobType = Object.keys(analyticsJobConfig.analysis)[0] as DataFrameAnalyticsType;
```

Comment:
We have `getAnalysisType()` available in `plugins/ml/public/application/data_frame_analytics/common/analytics.ts`.

Reply:
Updated here https://github.com/elastic/kibana/pull/76735/commits/cc8f5dd3c6567646ceea67439dd8544e8fb7789d